sysutils/freeradius/Portfile

```diff
@@ -27,24 +31,24 @@ long_description        The FreeRADIUS Server Project is an attempt to create \
                         RADIUS server, but they don't share a lot in common \
                         anymore.
 
-master_sites            ftp://ftp.freeradius.org/pub/freeradius/ \
-                        ftp://ftp.freeradius.org/pub/freeradius/old
+openssl.branch          1.1
 
-depends_lib             port:libtool \
+depends_lib-append      port:libtool \
                         port:libpcap \
                         path:lib/libldap.dylib:openldap \
-                        path:lib/libssl.dylib:openssl \
                         port:talloc \
-                        port:gdbm
-
-use_autoconf            yes
+                        port:gdbm   \
+                        port:pcre   \
+                        port:perl${perl5.default_branch}
```

Comment:
I’m not sure if it is okay to depend on the default Perl version this way. The only other port I’m aware of with this dependency behavior is `stow`, and it does so through the perl5 portgroup—maybe that isn’t okay either. The problem I predict with this is that if a different version variant of `perl5` is installed, then the port will suddenly depend on a different Perl version without its revision having been incremented, which for `freeradius` leads to a broken library link (at least when installing from the binary rather than building locally). It seems like manually specifying and updating the default branch in ports is a necessary chore to prevent breakage.

It should be fine to instead use:

```suggestion
                        port:perl${perl5.major}
```

and put `perl5.branches 5.28` (since others insist ports not use newer Perl yet) somewhere before this.

Reply:
https://trac.macports.org/ticket/64079 describes this issue.

Reply:
Perl 5.34 is now default (#15239)